src/Microsoft.Framework.Configuration.Json/Json/JsonConfigurationFileParser.cs

```diff
@@ -101,13 +101,13 @@ private void VisitPrimitive(JToken data)
         private void EnterContext(string context)
         {
             _context.Push(context);
-            _currentPath = string.Join(":", _context.Reverse());
+            _currentPath = string.Join(Constants.KeyDelimiter[0], _context.Reverse());
```

Comment:
@victorhurdugaci the the ```KeyDelimiter``` I defined in the PR is of type ```string[]```, so ```Constants.KeyDelimiter[0]``` will return a string :smile: